### PR TITLE
bugfix/search

### PR DESCRIPTION
**Pull request by iuriifaevskii**
* fixed functional switching type of search (search by ingredients, search by tags)
* fixed styles recipe-card and search component for the Mozilla Firefox

### CODE REVIEW

File: static/js/pages/RecipesPage/SearchComponent/SearchComponent.js [hunk 1]
```diff
@@ -67,58 +67,62 @@ class SearchComponent extends Component {
         this.setState({ elements: [] });
     }
 
+    requestToSearch(item, searchParam) {
+        wait(200)
```
Comment (apasashkov) [inline static/js/pages/RecipesPage/SearchComponent/SearchComponent.js]: why do we need wait here?

File: static/js/pages/RecipesPage/SearchComponent/SearchComponent.js [hunk 2]
```diff
@@ -67,58 +67,62 @@ class SearchComponent extends Component {
...
-        this.setState({ type });
-        this.setState({ elements: [] });
+        this.setState({ type, elements: [] });
     }
 
     render() {
         const phase = this.state.process;
+        const type = this.state.type;
         let searchElement = null;
+        let inputSearch = null;
```
Comment (apasashkov) [inline static/js/pages/RecipesPage/SearchComponent/SearchComponent.js:109]: what is the difference between searchElement and inputSearch and why we need 2 variables?

File: static/js/pages/RecipesPage/SearchComponent/SearchComponent.js [hunk 3]
```diff
@@ -127,7 +131,7 @@ class SearchComponent extends Component {
                         <form onSubmit={this.onSubmit}>
```
Comment (apasashkov) [inline static/js/pages/RecipesPage/SearchComponent/SearchComponent.js]: we agreed with Misha to put Dropdown outside of the form

File: static/js/pages/RecipesPage/SearchComponent/SearchComponent.js (hunk 2 above)
Comment (iuriifaevskii) [reply static/js/pages/RecipesPage/SearchComponent/SearchComponent.js:109]: two different components are displayed at the same time. There can't be one variable

**Review by apasashkov — APPROVED** [review]

**Pull request merged**